libraries/AP_BattMonitor/AP_BattMonitor_Backend.cpp

```diff
@@ -214,4 +216,28 @@ void AP_BattMonitor_Backend::check_failsafe_types(bool &low_voltage, bool &low_c
     } else {
         low_capacity = false;
     }
+
+    // check battery cell voltages
+    if (has_cell_voltages()) {
+
+        // critical cell voltage
+        if ((_params._crt_cell_voltage > 0) && (_state.cell_min < _params._crt_cell_voltage)) {
+            critical_cell_voltage = true;
+        } else {
+            critical_cell_voltage = false;
+        }
+
+        // low cell voltage or highest to lowest diff too great
```

Comment:
Cell spread should not overload the low voltage one, as it hides information from the caller/makes the overall display of failures more problematic.

Reply:
Still can't think of a better way that isn't a mess.  It won't override anything else. But I agree there must be a better way. And this kinda also goes back to the status text not being able to convey the full message.